docs/intro/reusable-apps.txt

```diff
@@ -288,6 +288,70 @@ working. We'll now fix this by installing our new ``django-polls`` package.
 
     pip uninstall django-polls
 
+Testing your app
+================
+Since we moved the ``polls`` directory out of the project,
+we don't have the project's ``settings.py`` and ``urls.py`` anymore.
+We need another way to configure our app in order to test it.
+
+#. First we create a ``tests`` module with an empty ``__init__.py`` and move
+   the tests over from the ``polls`` directory. Because the tests now reside
+   in a different package we will have to replace the relative import
+   ``from .models import Poll`` with ``from polls.models import Poll``.
+
+#. Now we create a ``settings.py`` inside the ``tests`` folder which will contain
+   only the settings required to test our application.
+
+   .. code-block:: python
+       :caption: tests/settings.py
+
+       from django.urls import include, path
+
+       SECRET_KEY = 'fake-key'
+
+       INSTALLED_APPS = [
+           "polls",
+           # adding tests allows us to define models inside the tests package
+           # "tests",
+       ]
+
+       DATABASES = {
+           'default': {
+               'ENGINE': 'django.db.backends.sqlite3',
+               'NAME': ':memory:',
+           }
+       }
+
+       TEMPLATES = [
+           {
+               'BACKEND': 'django.template.backends.django.DjangoTemplates',
+               'APP_DIRS': True,
+           },
+       ]
+
+       ROOT_URLCONF = 'tests.urls'
```

Comment:
Maybe you can document the pytest alternative, create a conftest.py file as such:

```
from django.conf import settings    
    
    
def pytest_configure():    
    settings.configure(    
        INSTALLED_APPS=[    
            'django.contrib.contenttypes',    
            'django.contrib.auth',    
        ],    
        DATABASES=dict(default=dict(    
            ENGINE='django.db.backends.sqlite3',    
            NAME=':memory:',    
        ))    
    )    
```